### ISSUE TITLE
Improving Breadcrumb accessibility

### ISSUE THREAD

**Issue by carmacleod** [carbon-design-system/carbon#1608]
Please see the [APG pattern for Breadcrumb](http://w3c.github.io/aria-practices/examples/breadcrumb/index.html) and consider revising the [Carbon Breadcrumb](https://www.carbondesignsystem.com/experimental/breadcrumb/code) in the following ways (that are shown in the APG example):
- use a list (_ordered_ list is the most semantically correct, because the order of the links matters) to contain the links (use css to hide the bullets and make the list horizontal). The advantage of this is that a screen reader can tell the user how many links are in the breadcrumb (i.e. it counts the items in the list. It would not count the children of a div because there is no semantic reason to do so).
- add `aria-current="page"` to the final link in the breadcrumb, so that screen readers can tell the user that this is the breadcrumb's current page.

**Comment by joshblack** [comment]
cc @dakahn

**Comment by dakahn** [comment]
Hi @carmacleod 👋🏼 Great great suggestions. Thanks for pointing out the issues with the current breadcrumb. I'll assign this to myself.

**Comment by snidersd** [comment]
@dakahn Also be sure the breadcrumb is in a navigation landmark region. The latest version in the Carbon sandbox is in main and it should be nav and have an aria-label="breadcrumb".

**Comment by dakahn** [comment]
v10 breadcrumb is still a valid candidate for a refactor into an ordered list. keeping this open
